Dockerfile

```diff
@@ -13,7 +17,39 @@ RUN go get -d ./...
 
 RUN CGO_ENABLED=1 go build "$GO_BUILD_ARGS" -o ./build/_output/bin/dynatrace-operator ./cmd/operator/
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi-micro:8.4
+
+COPY --from=k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0 /csi-node-driver-registrar /usr/local/bin
+COPY --from=k8s.gcr.io/sig-storage/livenessprobe:v2.5.0 /livenessprobe /usr/local/bin
+
+# copy tools required by init.sh
+COPY --from=package-download /usr/bin/unzip /usr/local/bin/unzip
+COPY --from=package-download /usr/bin/curl /usr/local/bin/curl
+
+# copy curl dependencies
+COPY --from=package-download /etc/pki/tls/certs /etc/pki/tls/certs
+COPY --from=package-download /etc/pki/ca-trust/extracted /etc/pki/ca-trust/extracted
+COPY --from=package-download /usr/lib64/libcurl.* \
+     /usr/lib64/libssl.* \
+     /usr/lib64/libcrypt* \
+     /usr/lib64/libz.* \
+     /usr/lib64/libnghttp2.* \
+     /usr/lib64/libidn2.* \
+     /usr/lib64/libssh.* \
+     /usr/lib64/libpsl.* \
+     /usr/lib64/libgssapi_krb5.* \
+     /usr/lib64/libkrb5.* \
+     /usr/lib64/libk5crypto.* \
+     /usr/lib64/libcom_err.* \
+     /usr/lib64/libldap-* \
+     /usr/lib64/liblber-* \
+     /usr/lib64/libbrotlidec* \
+     /usr/lib64/libunistring.* \
+     /usr/lib64/libkrb5support.* \
+     /usr/lib64/libkeyutils.* \
+     /usr/lib64/libsasl2.* \
+     /usr/lib64/libbz2.* \
+     /usr/lib64/libbrotlicommon.* /usr/lib64/
```

Comment:
How would this look like if curl is replaced by wget?

Reply:
Would look similar, we would just have a different set of dependencies and in addition have to change the commands in our init.sh

One solution would be a statically linked curl, but haven't found a trustworthy source for such a build.

Therefore, if we don't want to build our own statically linked curl, I would stick with this solution.